apps/storybook-react/stories/Table.stories.tsx

```diff
@@ -122,3 +131,42 @@ export const FixedTableHeader: Story<TableProps> = () => {
     </FixedContainer>
   )
 }
+
+export const CompactTable: Story<TableProps> = () => {
+  const data = [
+    { number: '1', name: 'Banana', colour: 'Yellow' },
+    { number: '2', name: 'Orange', colour: 'Orange' },
+    { number: '4', name: 'Kiwi', colour: 'Greenish' },
+  ]
+  return (
+    <Table density="compact">
```

Comment:
Did you consider `<Table compact>` instead of `<Table density="compact">` since there are only two possible values? 🤔 Also since `compact` is the exception to the rule 🤷🏻

Reply:
We considered using just `compact`, but felt it lacking in terms of future compatibility and component api design. Part of discussion is here: #823. 

If someone has `compact` as default for their application, then `comfortable` would be the exception to that applications rule. Having a `density` prop also makes it easier for users to implement a global toggle for setting density application wide.